exercises/concept/currency-exchange/.docs/instructions.md

```diff
@@ -61,8 +65,9 @@ This function should return the *number of bills* that you can get using the *bu
 
 Create the `exchangeable_value()` function, taking `budget`, `exchange_rate`, `spread`, and `denomination`.
 
-Parameter `spread` is the *percentage taken* as an exchange fee.
-If `1.00 EUR == 1.20 USD` and the *spread* is `10`, the actual exchange will be: `1.00 EUR == 1.32 USD`.
+Parameter `spread` is the *percentage taken* as an exchange fee, written as an integer. 
+It needs to be converted to decimal by dividing it by 100. 
+If `1.00 EUR == 1.20 USD` and the *spread* is `10`, the actual exchange rate will be: `1.00 EUR == 1.32 USD` because 10% of 1.20 is 0.12, and this additional fee is added to the exchange.
```

Comment:
For the time being this is fine. _But_ we are not here to explain basic math to all students. It's important for people to know how to deal with basic `text exercises`.

As soon as I find a good topic to rewrite this exercise, I will fix 🌟 all this 🌟 .

Reply:
💜   _And you will get aaaallll the sparkle points for rewriting!!_  I agree we aren't here to explain how to break down word problems, or how to deal with general math problems.  At some point, we _may_ want to do some short pieces or vids or walk-throughs on word-problem strategies.....but later.  😄